geary-commons-papermc/src/main/kotlin/com/mineinabyss/geary/minecraft/properties/ConfigurableLocation.kt

```diff
@@ -19,7 +19,7 @@ import org.bukkit.entity.Player
  * Useful for actions that wish to run at a specific location but want to remain configurable.
  */
 @Serializable
-public sealed class ConfigurableLocation : EntityProperty<Location>()
+sealed class ConfigurableLocation : EntityProperty<Location>()
```

Comment:
Why did you remove the public from these classes?

Reply:
Just said "redundant" in warnings when comitting so poofed shrug